Ctfl-v4-chapter3-static-testing.html

```
@@ -0,0 +1,1650 @@
+<meta charset="utf-8" />
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)